syntax = "proto3";

option go_package = "../common";
import "google/protobuf/empty.proto";

message EmailData {
  int32 label = 1;
  string text = 2;
}

message EmailResponse {
  string message = 1;
}

service EmailService  {
  rpc InsertEmail (EmailData) returns (EmailResponse);
  rpc DeleteAllEmail (google.protobuf.Empty) returns (EmailResponse);
}